From: David Faure <faure@kde.org>
Date: Thu, 02 Jan 2014 00:58:01 +0000
Subject: re-enable linking to KF5::NotifyConfig, otherwise the header is not found
X-Git-Url: http://quickgit.kde.org/?p=kde-workspace.git&a=commitdiff&h=ad6c6f9a690efa13253a8e4e2b31e92dca9a0fe8
---
re-enable linking to KF5::NotifyConfig, otherwise the header is not found
---


--- a/plasma/generic/dataengines/notifications/CMakeLists.txt
+++ b/plasma/generic/dataengines/notifications/CMakeLists.txt
@@ -18,7 +18,7 @@
     KF5::Notifications
     KF5::Plasma
     KF5::Service
-    #KF5::NotifyConfig
+    KF5::NotifyConfig
 )
 
 install(TARGETS plasma_engine_notifications DESTINATION ${PLUGIN_INSTALL_DIR}/plasma/dataengine)

--- a/plasma/generic/dataengines/notifications/notificationsengine.cpp
+++ b/plasma/generic/dataengines/notifications/notificationsengine.cpp
@@ -297,8 +297,7 @@
 
 void NotificationsEngine::configureNotification(const QString &appName)
 {
-#warning "reenable once KNotifyConfigWidget works again"
-//    KNotifyConfigWidget::configure(0, appName);
+    KNotifyConfigWidget::configure(0, appName);
 }
 
 K_EXPORT_PLASMA_DATAENGINE_WITH_JSON(notifications, NotificationsEngine, "plasma-dataengine-notifications.json")

